_posts/2020-05-26-exploring-the-open-sgid-a-brief-tutorial-using-open-source-python-tools.md

```diff
@@ -92,9 +104,13 @@ plt.show()
 
 [![Davis County Address Points]({% link images/EMN_20200526_addpts_plot.png %} "Davis County Address Points")]({% link images/EMN_20200526_addpts_plot.png %}){: .flex .flex--center }
 
-### Utah Faults Longer than One Mile
+### EXAMPLE 3: Utah Faults Longer than One Mile
 
-In the third example, we'll demonstrate the subsetting capabilities of Pandas/GeoPandas to find and plot fault lines in Utah that are longer than 1 mile (1609.34 meters).  First, we'll specify the layer we want to work with in our SQL query.  Second, we'll create a subset of all the faults (called 'long_faults') by selecting those where the 'shape' field has a length greater than 1609.34 meters.  We use `.copy()` at the end of this line of code to prevent a 'chained assignment' warning in Pandas.  Next, we'll calculate a new field in this subset that converts the length into miles, for easier interpretation.  Finally, we plot the counties and faults longer than 1 mile, colored by length.  Note that the syntax for the legend is a little different now that we are plotting a continuous variable (lengths a floating point) rather than a discrete variable (zipcodes as strings).  We also specify the starting and ending ranges for the colorbar (`vmin` and `vmax`).
+This third example demonstrates the subsetting capabilities of Pandas/GeoPandas to find and plot fault lines in Utah that are longer than 1 mile (1609.34 meters).
```

Comment:
**question**: do we want to call out why we convert miles to meters here? I think it is important to understand our data is stored as meters. I'm not sure how common knowledge that is.

Reply:
I'll add this sentence: "Since Open SGID data is in meters, a conversion to miles is done to improve interpretation."

Reply:
Updated to: "The native unit of Open SGID data is meters, so calculations and queries are performed in meters.  After performing the query, a conversion to miles is done to demonstrate how to add a field in GeoPandas and to improve human interpretation of the threshold used."